### ISSUE TITLE
Add a verbose mode for easier troubleshooting and log error messages

### ISSUE THREAD

**Issue by nitsanw** [jrudolph/perf-map-agent#34]
Currently PMA fails or succeeds silently. When things are good this is a non-issue, when something doesn't work you probably want to see a bit more output.
I've played around with the idea and would like to log:

 1. Any JVMTI failure
 2. Agent method processing time
 3. At debug level add a per method printout of some form?

Ideas and feedback before I turn this into a PR?

**Comment by jrudolph** [comment]
Yes, all of these points certainly makes sense. In #35, I will add a debug option to output full inline info to the console but this can certainly somewhat improved.
